src/box/relay.cc

```diff
@@ -987,12 +1044,21 @@ relay_subscribe_f(va_list ap)
 	if (!relay->replica->anon)
 		trigger_add(&relay->r->on_close_log, &on_close_log);
 
+	/*
+	 * When recovering from a long WAL, fiber_yield() will occur after
+	 * N rows and the current fiber will be called.
+	 */
+	char name[FIBER_NAME_MAX];
+	snprintf(name, sizeof(name), "%s:%s", fiber()->name, "uptime");
```

Comment:
Second `%s` can be just replaced with `uptime`.

Reply:
Fixed.